src/pages/Landing/index.js

```diff
@@ -9,7 +9,7 @@ const Landing = () => {
   const [showMessage, setMessage] = useState('');
 
   const formSwitch = param => {
-    switch(param) {
+    switch (param) {
```

Comment:
It's likely we can just turn this into a ternary.

Reply:
agreed